src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -228,11 +231,34 @@ module.exports.external = [
         }
     },
     {
-        name: 'base64Encode',
-        description: 'Returns base64 encoded string: base64Encode string',
+        name: 'contains',
+        description: 'Returns true if a string includes another string: contains parentStr childStr',
+        func: function (parentStr, childStr) {
+            if (!parentStr) {
+                return false;
+            }
+            return parentStr.toString().includes(childStr);
+        }
+    },
+    {
+        name: 'sha1Hash',
```

Comment:
Should we be using sha1 since it is insecure?

Reply:
This is as per spec : https://www.hl7.org/fhir/datatypes.html#Attachment